orgs/beta_prerelease.json

```diff
@@ -5,10 +5,16 @@
   "hasSampleData": "true",
   "settings": {
     "orgPreferenceSettings": {
-      "chatterEnabled": true,
-      "disableParallelApexTesting": true,
+      "chatterSettings": {
+        "enableChatter": true
+      },
+      "apexSettings": {
+        "enableDisableParallelApexTesting": true
+      },
       "notesReservedPref01": true,
```

Comment:
@jofsky  This one needs tweaking - convert `notesReservedPref01` and remove the `orgPreferenceSettings` header.

Reply:
Sorry bout that will fix up